.github/workflows/ci.yml

```
@@ -108,7 +109,7 @@ jobs:
       uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32
       if: ${{ always() }}
       with:
-        name: logs_func_win${{ matrix.windows }}_${{ matrix.configuration }}_${{ matrix.platform }}
+        name: logs_func_${{ matrix.apimode }}_win${{ matrix.os }}_${{ matrix.configuration }}_${{ matrix.platform }}
```

Just a nit: This whole repo is **win**-net-test. I feel like the Windows part is then implied for everything else. So it's unnecessary to include it in any output.
```suggestion
        name: logs_func_${{ matrix.apimode }}_${{ matrix.os }}_${{ matrix.configuration }}_${{ matrix.platform }}
```